src/MinimalDelegation.sol

```diff
@@ -124,20 +150,21 @@ contract MinimalDelegation is IERC7821, ERC1271, EIP712, ERC4337Account, KeyMana
         return MinimalDelegationStorageLib.get().entryPoint;
     }
 
-    function _isValidSignature(bytes32 _hash, bytes calldata _signature)
+    /// @notice Verifies that the key signed over the digest
+    /// Handles signatures from the root ECDSA key and wrapped signatures
```

Comment:
nit this assumes signatures are not wrapped so this is a little misleading, think u can just remove